### PR TITLE
Added snmpv1 traps

### PR DESCRIPTION
**Pull request by VenkateshRavula**
### Description
Added snmpv1 traps

### Check List
- [x] New functionality includes sanity testing.
  - [x] All sanity tests pass. (`$ ansible-test sanity`).
- [x] New functionality has been documented in the README if applicable.
  - [x] New functionality has been thoroughly documented in the examples (please include helpful comments).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 # Ansible Collections for HPE OneView Change Log
 
+## v6.0.0(unreleased)
+This release extends the planned support of the collections to OneView REST API version 2600 (OneView v6.0)
+
+### Modules supported in this release
+- oneview_appliance_device_snmp_v1_trap_destion
```
Comment (SHANDCRUZ) [inline CHANGELOG.md:7]: *destination

File: tests/unit/test_oneview_appliance_device_snmp_v1_trap_destinations.py [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+###
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+import pytest
+
+from hpe_test_utils import OneViewBaseTest
+from oneview_module_loader import ApplianceDeviceSnmpV1TrapDestinationsModule
```
Comment (SHANDCRUZ) [inline tests/unit/test_oneview_appliance_device_snmp_v1_trap_destinations.py:24]: ansible_collections.hpe.oneview.tests.unit.utlis. not included

**Review by SHANDCRUZ — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (VenkateshRavula) [reply CHANGELOG.md]: done

File: tests/unit/test_oneview_appliance_device_snmp_v1_trap_destinations.py (hunk 2 above)
Comment (VenkateshRavula) [reply tests/unit/test_oneview_appliance_device_snmp_v1_trap_destinations.py]: done

**Review by AsisBagga — APPROVED** [review]
LGTM

**Pull request merged**